llvm/test/CodeGen/X86/vector-reduce-fmax-nnan.ll

```diff
@@ -413,9 +413,12 @@ define half @test_v2f16(<2 x half> %a0) nounwind {
 ; AVX512F:       # %bb.0:
 ; AVX512F-NEXT:    # kill: def $xmm0 killed $xmm0 def $zmm0
 ; AVX512F-NEXT:    vpsrld $16, %xmm0, %xmm1
-; AVX512F-NEXT:    vcvtph2ps %xmm0, %ymm2
-; AVX512F-NEXT:    vcvtph2ps %xmm1, %ymm3
-; AVX512F-NEXT:    vcmpltps %zmm2, %zmm3, %k1
+; AVX512F-NEXT:    vcvtph2ps %xmm0, %xmm2
+; AVX512F-NEXT:    vcvtph2ps %xmm1, %xmm3
+; AVX512F-NEXT:    vucomiss %xmm3, %xmm2
+; AVX512F-NEXT:    seta %al
+; AVX512F-NEXT:    negb %al
+; AVX512F-NEXT:    kmovd %eax, %k1
```

Comment:
Annoying but not that important

Reply:
Yeah, but we don't need to tune much for unpractical combination.